### PR TITLE
Aims 335 add worker permissions

### PR DESCRIPTION
**Pull request by BiigY**
I need a review of my code, to be more exact, I updated some controller files in Annex project, as well as wrote some spec unit testing. All changes were pushed already. I would like to see any feedback and suggestions.

### CODE REVIEW

File: app/views/users/index.html.haml [hunk 1]
```diff
@@ -34,3 +34,5 @@
           window.table = $('#users').DataTable();
           $('div.dataTables_filter input').focus();
         } );
+- else
```
Comment (jgondron) [inline app/views/users/index.html.haml]: Is this left over from the if admin check that you removed?

File: config/initializers/devise.rb [hunk 2]
```diff
@@ -4,7 +4,7 @@
   # The secret key used by Devise. Devise uses this key to generate
   # random tokens. Changing this key will render invalid all existing
   # confirmation, reset password and unlock tokens in the database.
-  # config.secret_key = '011b6001f75749566e092721113095fc81cef7c845b0854f0e3280861558b976744c6d080c9f921a7a07bc251d1a07ce558d425ce92b80f1e5ec3e519b001f7d'
+  config.secret_key = "011b6001f75749566e092721113095fc81cef7c845b0854f0e3280861558b976744c6d080c9f921a7a07bc251d1a07ce558d425ce92b80f1e5ec3e519b001f7d"
```
Comment (jgondron) [inline config/initializers/devise.rb]: Can you double check to see if this was necessary?

**Comment by jgondron** [comment]
:+1:  after hound comments are addressed

**Comment by rfox2** [comment]
This looks good Yerlan! The hound comments need to be addressed - those are code style issues that we try to adhere to. After that, :+1: and you can merge this with master. Make sure the Jira ticket then gets moved over into QA.

**Pull request merged**